.github/workflows/critical-issue.yml

```diff
@@ -0,0 +1,39 @@
+name: Slack notification for critical issue
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  slack-notify:
+    name: Send slack notification if issue is labeled as critical
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'severity/critical' }}
+    steps:
+      - id: slack-notify
+        name: Send slack notification
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          # For posting a rich message using Block Kit
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":alarm: New critical issue! :alarm:"
+                  }
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Please check the issue: ${{ github.event.issue.url }}\n \\cc @zeebe-process-automation-team @zeebe-distributed-platform-team "
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
💭 this will post to #zeebe-ci , right? 🤔 would like to put is elsewhere like #team-c8-core-automation but this will require new secret setup. Let's get started with this first then.

Reply:
I agree, we need to add a web hook to Zeebot. I don't have permissions to do this sadly. It seems Nicolas, Philipp, Chris, Nico and Deepthi do have them.